x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/delete_rules_route.ts

```diff
@@ -43,9 +46,19 @@ export const createDeleteRulesRoute: Hapi.ServerRoute = {
         id,
         ruleId,
       });
-
+      const ruleStatuses = await savedObjectsClient.find<IRuleSavedAttributesSavedObjectAttributes>(
+        {
+          type: ruleStatusSavedObjectType,
+          perPage: 5,
+          search: `${rule?.id}`,
```

Comment:
Move this block below the if check and remove the `?` here for a `rule.id` with no string interpolation